openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetailPage.tsx

```diff
@@ -0,0 +1,172 @@
+import Description from '../../../components/common/description/Description';
+import RichTextEditorPreviewer from '../../../components/common/rich-text-editor/RichTextEditorPreviewer';
+import TitleBreadcrumb from '../../../components/common/title-breadcrumb/title-breadcrumb.component';
+import Loader from '../../../components/Loader/Loader';
+import {
+  GlobalSettingOptions,
+  GlobalSettingsMenuCategory,
+} from '../../../constants/globalSettings.constants';
+import { Policy } from '../../../generated/entity/policies/policy';
+import { EntityReference } from '../../../generated/type/entityReference';
+import { getEntityName } from '../../../utils/CommonUtils';
+import { getRoleWithFqnPath, getSettingPath } from '../../../utils/RouterUtils';
+import { showErrorToast } from '../../../utils/ToastUtils';
+import './PoliciesDetail.less';
+
+const { TabPane } = Tabs;
+
+const PoliciesDetailPage = () => {
+  const { fqn } = useParams<{ fqn: string }>();
+
+  const [policy, setPolicy] = useState<Policy>({} as Policy);
+  const [isLoading, setLoading] = useState<boolean>(false);
+
+  const breadcrumb = useMemo(
+    () => [
+      {
+        name: 'Policies',
+        url: getSettingPath(
+          GlobalSettingsMenuCategory.ACCESS,
+          GlobalSettingOptions.POLICIES
+        ),
+      },
+      {
+        name: fqn,
+        url: '',
+      },
+    ],
+    [fqn]
+  );
+
+  const fetchPolicy = async () => {
+    setLoading(true);
+    try {
+      const data = await getPolicyByName(fqn, 'owner,location,teams,roles');
+      setPolicy(data ?? ({} as Policy));
+    } catch (error) {
+      showErrorToast(error as AxiosError);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  const RolesList = ({ roles }: { roles: EntityReference[] }) => {
+    const columns: ColumnsType<EntityReference> = useMemo(() => {
+      return [
+        {
+          title: 'Name',
+          dataIndex: 'name',
+          width: '200px',
+          key: 'name',
+          render: (_, record) => (
+            <Link
+              className="hover:tw-underline tw-cursor-pointer"
+              to={getRoleWithFqnPath(record.fullyQualifiedName || '')}>
+              {getEntityName(record)}
+            </Link>
+          ),
+        },
+        {
+          title: 'Description',
+          dataIndex: 'description',
+          key: 'description',
+          render: (_, record) => (
+            <RichTextEditorPreviewer markdown={record?.description || ''} />
+          ),
+        },
+      ];
+    }, []);
+
+    return (
+      <Table
+        className="roles-list-table"
+        columns={columns}
+        dataSource={roles}
+        pagination={false}
+        size="middle"
+      />
+    );
+  };
+
+  useEffect(() => {
+    fetchPolicy();
+  }, [fqn]);
+
+  return (
+    <div data-testid="policy-details-container">
+      <TitleBreadcrumb titleLinks={breadcrumb} />
+      {isLoading && <Loader />}
```

Comment:
can we move this to the above return statement in if block
```suggestion
      if(isLoading){return <Loader />}
```
In this way we don't have to depend on isLoading condition in the components return statement
Let me know if you need any help over here